packages/p2p/src/pages/my-profile/block-user/block-user-empty/block-user-empty.tsx

```diff
@@ -27,9 +30,10 @@ const BlockUserEmpty = () => {
                 <MobileFullPageModal
                     body_className='block-user-empty'
                     height_offset='80px'
-                    is_modal_open={my_profile_store.active_tab === my_profile_tabs.MY_COUNTERPARTIES}
+                    is_modal_open={is_modal_open}
```

Comment:
**suggestion(non-blocking)**: we could also just inline the condition here directly, since the `is_modal_open` is not being reused anywhere else:
```suggestion
                    is_modal_open={my_profile_store.active_tab === my_profile_tabs.MY_COUNTERPARTIES}
```